rzk-playground/components/editor/Editor.tsx

```diff
@@ -19,12 +19,20 @@ export default function Editor({
     editorHeight: number
 }) {
     const [existsSelection, setExistsSelection] = useState(false)
+    const params = new URLSearchParams(window.location.search);
+
+    var snippet = example
+    if (params.has("snippet")) {
+        snippet = params.get("snippet")!;
+    } else if (params.has("code")) {
+        snippet = params.get("code")!;
+    }
```

Comment:
First, always use `let` instead of `var`. Second, I would simplify it to avoid the intermediate variable as such:
```suggestion
    if (params.has("snippet")) {
        setText(params.get("snippet")!);
    } else if (params.has("code")) {
        setText(params.get("code")!);
    } else {
        setText(example);
    }
```
preferable inside the same `useEffect` in which `params` should be defined.
Then you would also not need `setText(snippet)` inside `onCreateEditor`.